src/rustdoc-json-types/lib.rs

```diff
@@ -115,6 +115,35 @@ pub enum Visibility {
     },
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct DynTrait {
+    /// All the traits implemented. The first gives the vtable, and the rest must be auto traits.
+    pub traits: Vec<PolyTrait>,
+    /// The lifetime of the whole dyn object
+    /// ```plain
```

Comment:
```suggestion
    /// ```text
```

Reply:
Fixed for all occurrences in rustdoc-json-types